src/app.fetch-httpClient.config.js

```diff
@@ -45,6 +45,34 @@ export class FetchConfig {
         request.headers.append(config.authHeader, token);
 
         return request;
+      },
+      response(response, request) {
+        return new Promise(
+          (resolve, reject) => {
+            if (response.ok) {
+              resolve(response);
```

Comment:
But according to the aurelia-fetch api or my understanding of it is that the response interceptor has to return a Response Object or a Promise that resolves to a Response Object so i think it should be `return resolve(response)` instead